playwright-action/helpers/constants.js

```diff
@@ -0,0 +1,75 @@
+import { id } from "../helpers/playwrightSetup.js";
+
+export const DESIGNER = "designer"
+export const VISUALIZER = "visualizer"
+
+//Objects defining paths, aliases, and wait conditions for intercepting certain API requests.
+export const extension = {
+  path: "/api/provider/extension*",
+  alias: "extensionFileLoad"
+};
+
+export const designEndpoint = {
+  path: "/api/pattern*",
+  alias: "designEp",
+  wait: "@designEp",
+  absolutePath: "/api/pattern"
```

Comment:
Is `api/pattern` an endpoint?
What's the use of `designEndpoint`?

Reply:
@Philip-21, may you answer this?

Reply:
its the ApiEndpoint for  creating or deleting meshmap design .

Reply:
I see. Id added after it will point to a particular design, r8?

Reply:
yes

Reply:
i am thinking of writing individual tests for every action but i dont think it will work

Reply:
i am trying to see how i ca put everything together, even the login test in the spec.js file isnt working